sourcegraph/sourcegraph
# search: add suggestions for repo:deps predicate

Fixes #31897

## Test plan



* Go to any search input
* Type `repo:deps(` or `repo:dependencies(`
* You should get repository suggestions
